### DISCUSSION TITLE
MySQL TCP/IP over SSH

### DISCUSSION THREAD

**Discussion by enpro-github** [Budibase/budibase#3082] [Help]
I connect to a MySQL database via Standard TCP/IP over SSH.  Is it possible to connect this database to Budibase using the same connection method?

Thank you

**Comment by mjashanks** [comment]
yes - budibase supports `ssh`. However, I would flag this issue with you: https://github.com/Budibase/budibase/issues/1899

We do not currently support MySQL 8 - but are going to fix this asap.

**Reply by enpro-github** [reply]
Thanks for the quick reply

**Reply by enpro-github** [reply]
I still can't connect to MySQL.  Our MySQL is on an AWS RDS, which is not public.  We ssh into an EC2 and use portforwarding to connect to the RDS.  How can we do that with the budibase MySQL connector?

**Reply by shogunpurple** [reply]
Hi @enpro-github - our MySQL connector does not support connections over `ssh` natively. We have SSL support but it will not open up a tunnel for you. If you just need to connect, you can set up your own SSH tunnel and connect normally, but that's not a long term solution. Converting this into an issue

**Reply by enpro-github** [reply]
@shogunpurple Thank you
